lectures/software_engineering/testing.md

```diff
@@ -435,8 +435,8 @@ module MyProject
 
 using Expectations, Distributions
 
-function foo(μ = 1., σ = 2.)
-    d = Normal(μ, σ)
+function foo(;mu = 1., sigma = 2.)
```

Comment:
I would turn this back to `foo(mu = 1., sigma = 2.)`

YOu don't want to use keyword only functions very often.  Just for those convenience constructors and occiaonlly a few other things.